phpunit.xml.dist

```diff
@@ -30,4 +30,8 @@
         </whitelist>
     </filter>
 
+    <php>
+        <ini name="precision" value="8"/>
+    </php>
+
```

Comment:
hmm IMO we have to rewrite the other test (i mean the failing example), what do you think @core23